mono/metadata/mono-basic-block.h

```diff
@@ -31,8 +31,21 @@ mono_basic_block_free (MonoSimpleBasicBlock *bb);
 
 /*This function is here because opcodes.h is a public header*/
```

Comment:
This comment is a bit orphaned now.  I think it should go with `mono_opcode_value_and_size`, below.